src/GitTools.jl

```diff
@@ -103,8 +103,15 @@ function normalize_url(url::AbstractString)
     end
 end
 
-ensure_clone(ctx, target_path, url; kwargs...) =
-    ispath(target_path) ? LibGit2.GitRepo(target_path) : GitTools.clone(ctx, url, target_path; kwargs...)
+function ensure_clone(ctx, target_path, url; kwargs...)
+    if ispath(target_path)
+        @debug "Using existing repo"
```

Comment:
This seems like it needs a bit more context if it is going to be useful in general. Perhaps it is just for intermediate debugging.

Reply:
Yeah I'll just get rid of this for now